src/config.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+    baseURL: 'https://dev.xikolo.de',
+    kapacitorProxyURL: 'http://82.140.0.78:8082',
+    platforms: ['vwgroup-staging', 'codeocean', 'codeocean-staging', 'teambuilder', 'openwho', 'openhpi-staging',
```

Comment:
We should definitely not expose this to the public (github). is there another way to get these platforms through an api?